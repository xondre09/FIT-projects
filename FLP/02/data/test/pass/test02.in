S   B R
B a C R
C b D R
C   X  
X   E L
D e D R
D a B R
E a G  
E b G  
E   H R
G   E L
H   I y
I y F L
 abaa
