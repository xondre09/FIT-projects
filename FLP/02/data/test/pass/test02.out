S abaa
 Babaa
 aCbaa
 abDaa
 abaBa
 abaaC 
 abaaX 
 abaEa
 abaG 
 abEa
 abG 
 aEb
 aG 
 Ea
 G 
E 
 H 
 Iy
F y
